docs-client/README.md

```diff
@@ -18,21 +18,25 @@ $ ./gradlew :docs-client:yarn_run_start --no-daemon
 ```
 
 This will usually not be useful since without a server running, the client does not have any spec it can render.
-You can have server calls proxied to a running Armeria server by specifying the `PROXY_PORT` environment
-variable, e.g.,
+You can have server calls proxied to a running Armeria server by specifying environment variables.
+
+* `PROXY_PORT`: Armeria server port (default: 8080)
```

Comment:
How about renaming `PROXY_PORT` to `ARMERIA_PORT`? I realized that I was a little bit confused whether the `PROXY_PORT` means the port of Armeria server or the proxy server. /cc @anuraaga